src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -840,6 +843,8 @@ maybe_convert_reductions({KVs0, UserReductions}) ->
 
 maybe_convert_kv({<<"_local/", _/binary>> = DocId, _}) ->
```

Comment:
Its an upgrade clause for the time being. The #doc version will already be caught on line 846.